config/v1/types_ingress.go

```diff
@@ -54,6 +54,14 @@ type IngressSpec struct {
 	// configurable routes.
 	// +optional
 	ComponentRoutes []ComponentRouteSpec `json:"componentRoutes,omitempty"`
+
+	// requiredRouteAnnotations is an optional list of default annotations
```

Comment:
indicate how order is handled and how it can be used to exclude `openshift-*` namespaces.